redis/CHANGELOG.md

```diff
@@ -1,9 +1,43 @@
+<a name="0.22.0"></a>
+### 0.22.0 (2022-09-30)
+
+#### Features
```

Comment:
I like to also call out external contributors (that is, every PR author who is not you or me), how do you feel about that?

Reply:
Done.